### DISCUSSION TITLE
我们可以自己整理一些关于电机学的资料

### DISCUSSION THREAD

**Discussion by hh-wu** [POLYU-EMLAB/polyu-emhub#3] [Ideas]
很多内容涉及版权，索性我们自己重新改写一遍，也有利于学习。尤其对于新来的同学们。

参考[费曼技巧](https://www.zhihu.com/question/20576786)

**Comment by jiangmy97** [comment]
目前上传了一些放在此处[https://github.com/POLYU-EMLAB/useful-resources](https://github.com/POLYU-EMLAB/useful-resources)
大家有新的教程之类也敬请上传。
